paasta_tools/marathon_tools.py

```diff
@@ -255,7 +256,7 @@ def get_backoff_seconds(self):
         if instances == 0:
             return 1
         else:
-            return float(10.0 / instances)
```

Comment:
Tests were failing due to the API wanting an integer instead of a float: https://github.com/mesosphere/marathon/blob/cf504d13e0dac9f6886ed9bcac0513cb5c6b7390/docs/docs/rest-api/public/api/v2/schema/AppDefinition.json#L45